proxy/package.json

```diff
@@ -26,8 +26,8 @@
         "@openzeppelin/contracts-upgradeable": "^4.7.1",
         "@openzeppelin/hardhat-upgrades": "^1.9.0",
         "@skalenetwork/etherbase-interfaces": "^0.0.1-develop.20",
-        "@skalenetwork/ima-interfaces": "1.0.0-develop.24",
-        "@skalenetwork/skale-manager-interfaces": "1.0.0",
+        "@skalenetwork/ima-interfaces": "^1.1.0-develop.0",
```

Comment:
Looks like we can use stable version here

Reply:
Initially I tried it but we already have interface changes in `develop`.